TAO/tao/CDR.h

```diff
@@ -499,6 +499,8 @@ TAO_Export CORBA::Boolean operator<< (TAO_OutputCDR &os,
 TAO_Export CORBA::Boolean operator<< (TAO_OutputCDR &os,
                                       ACE_OutputCDR::from_std_wstring x);
 #endif /* ACE_LACKS_STD_WSTRING */
+TAO_Export CORBA::Boolean operator<< (TAO_OutputCDR &os, CORBA::Int8 x);
+TAO_Export CORBA::Boolean operator<< (TAO_OutputCDR &os, CORBA::Uint8 x);
```

Comment:
Are these going to result in ambiguous overload resolution?

Reply:
As opposed to `char`? They probably will in ACE6 depending on the backing type, but with `int8_t` and `uint8_t` they seem to be fine. I was thinking of ifdefing them out in ACE6 if the types weren't stdint-backed.

Reply:
int8_t is a typedef/alias of some other type -- since there's already `from_octet` and `from_char` in `ACE_OutputCDR` shouldn't we have the same for u/int8?

Reply:
Yea I wasn't thinking about this right. I will redo the serialization to make it like `octet`/`char`.